packages/database/src/server/changeHandlers/entityHierarchyCacher/EntityParentChildRelationBuilder.js

```diff
@@ -81,7 +88,8 @@ export class EntityParentChildRelationBuilder {
     await this.deleteObsoleteRelationsForParents(hierarchyId, parentIds, validParentChildIdPairs);
 
     const latestChildrenAlreadyCached = new Set([...childrenAlreadyCached, ...validChildIds]);
-    return this.fetchAndCacheChildren(hierarchyId, validChildIds, latestChildrenAlreadyCached);
+  
+    return await this.fetchAndCacheChildren(hierarchyId, validChildIds, latestChildrenAlreadyCached);
```

Comment:
Why the new `await` here?

Reply:
Oh it was a random attempt to fix a bug before, turns out it wasn't because of that. I'll remove it